README.MD

```diff
@@ -166,40 +173,86 @@ struct BatchedOutputOrders {
+
+## Nested Factory interaction with the Nested Reserve and Nested Records
+The Nested Reserve stores underlying assets of all NestedNFTs. The Nested Records keeps track of which underlying assets are associated with a specific NestedNFT. 
+Hence, each time the Nested Factory needs to interact with user funds (which are represented as a NestedNFT), it will first check the balance of tokens associated with the NestedNFT through Nested Records. If needed, it will then transfert funds to the Nested Reserve or withdraw funds from it.
+
+## ETH managment
+
+The Nested protocol only handles WETH when calling operators.  
+The sent ETH used to feed a portfolio is automatically converted to WETH when received.
+
+### WETH Conversion
+
+The conversion from ETH to WETH is done when submitting an order through `_submitInOrders` or `_submitOutOrders`.
+
+Before submitting orders, the NestedFactory transfers the input tokens from the NestedReserve (or the msg.sender) to the factory and converts the sent ETH to WETH.
+
+### Use of ETH in operators
+
+There are some operators who use ETH directly and not WETH.  
+In this case, the operator uses the Withdrawer to get the ETH back from the WETH contract before using it.
+
+> Only the Withdrawer can send ETH to the NestedFactory without automatic WETH conversion.
+
+## Royalties
+Royalties are withdrawal fees when a user has copied an existing portfolio and are managed by the `FeeSplitter.sol`. Royalties are distributed from the `NestedFactory.sol` when a user withdraws funds from his portfolio OR when an order fails, all tokens are returned to the sender and are therefore subject to withdrawal fees.
+
+### Fees distribution
+For now, protfolio designers and Nested share equally the royalties.
+This distribution is done in the `feeSplitter.sol` with the ratio between the `royaltiesWeight` and the `shareholders` weights.
+Currently, we have `royaltiesWeight = 50` and **one** shareholder (_Nested_) with `weight = 50`.
+
+> **totalWeights** always equals **royaltiesWeight** + all shareholders **weights**
+
 # Ownership & Governance
 
-Some functions of the protocol require admin rights (`onlyOwner`).
+Some functions of the protocol require admin rights (`onlyOwner` using `Ownable` from [OpenZeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol)). Same with the [TransparentUpgradeableProxy](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) which need an **Admin**.
 
-The contracts are owned by the [TimelockController](https://docs.openzeppelin.com/contracts/4.x/api/governance#TimelockController) contract from OpenZeppelin, set with a **7-days** delay.
-This ensures the community has time to review any changes made to the protocol.
+<img width="1011" alt="image" src="https://user-images.githubusercontent.com/22816913/173034995-432d6e84-e9d2-40cf-acb3-e109a158240f.png">
```

Comment:
I would suggest to upload the image to the directory static (name can be reviewed, `assets` maybe ?).
It will allow us to know from where the image is and have an history.  
You can then display the image like just above
`<div align="center"><img src="./static/output-orders.png" width="650"></div>`